src/button.js

```diff
@@ -29,7 +29,9 @@ function Button (command, opts) {
     this._disabled = false;
     this._label = opts.label || '';
     this._errback = opts.errback;
-    this._insightsVerb = opts.insightsVerb || opts.label || '';
+    this._insightsVerb = opts.insightsVerb === null ?
+        null :
+        opts.insightsVerb || opts.label || null;
```

Comment:
Why does this use `opts.label`? Seems like we're always passing in `opts.insightsVerb`, so it may not be needed? I would love to remove that so that we can do:
```
this._insightsVerb = opts.insightsVerb || null;
```
I don't like how we have to check if it's null in other places before passing along the value.